classes/person.rb

```diff
@@ -1,8 +1,11 @@
-class Person
+require_relative 'nameable'
+
+class Person < Nameable
```

Comment:
**(LoadError)**
![image](https://github.com/over-geek/OOP-School-Library/assets/78251927/63b14556-932b-48e1-acde-2f3dbf72550d)

Since the file `nameable` is outside the `classes` directory, I would suggest that we use `require` which is used to load external files. 

i.e.
```ruby 
require './nameable'
```

`require_relative` is usually used for referring to files within our directory.  🤓🛠

[Understanding Require vs. Require_relative vs. Require_all](https://medium.com/@ellishim/understanding-require-vs-require-relative-vs-require-all-80e3b26d89e6)